SU-HSDO/stanford_mrc
# MRCD8-216 homepage styles and tweaks

# READY FOR REVIEW

# Summary
- General homepage tweaks (more broad work is part on more specific tickets)

# Needed By (Date)
- End of sprint (2/15)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Verify that the MRC Spotlight postcard more closely resembles the mock, that there are vertical margin between cards (grid system FIXUP), that header has cloud-colored background...
3. More specifics can be seen on the ticket linked to below

# Affected Projects or Products
- stanford_mrc
- MRC

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-216

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
